kudos-board-ui/src/components/BoardCard/BoardCard.jsx

```diff
@@ -3,8 +3,10 @@ import axios from "axios";
 import "./BoardCard.css";
 
 function BoardCard({ board, onDelete }) {
+  // Move this outside of the component
```

Comment:
Let's move this outside the component so that it's not occurring within the react lifecycle.